Add note about blank payloads (#554)
FamilySearch/GEDCOM: specification/gedcom-3-structures-1-organization.md

    or in the `HEAD`.`PLAC`.`FORM` if there is no `FORM` substructure.
    If neither `FORM` exists, the meaning of the elements are not defined in this specification beyond being names of jurisdictions of some kind, ordered from smallest to largest.

:::note
    Existing applications often default to assuming a `FORM` of "City, County, State, Country",

>     and may even ignore any `FORM` substructures and treat payloads with a smaller number of

Author: ```suggestion
    and some even ignore any `FORM` substructures and treat payloads with a smaller number of
```

    elements as if they had additional blank elements at the end. Accordingly, to increase

>     the chances of interoperability it is recommended that blank elements be used to ensure

Author: ```suggestion
    the chances of interoperability an application might want to use blank elements to ensure
```

>     Existing applications often default to assuming a `FORM` of "City, County, State, Country",
>     and some even ignore any `FORM` substructures and treat payloads with a smaller number of
>     elements as if they had additional blank elements at the end. Accordingly, to increase
>     the chances of interoperability it is recommended that blank elements be used to ensure
>     payloads can be interpreted as best as possible in such a circumstance.

Author: ```suggestion
    Some applications or users have defaulted to assuming a `FORM` of "City, County, State, Country",
    and some even ignore any `FORM` substructures and treat payloads with a smaller number of
    elements as if they had additional blank elements at the end.
```

> :::note
>     Existing applications often default to assuming a `FORM` of "City, County, State, Country",
>     and some even ignore any `FORM` substructures and treat payloads with a smaller number of
>     elements as if they had additional blank elements at the end. Accordingly, to increase
>     the chances of interoperability it is recommended that blank elements be used to ensure
>     payloads can be interpreted as best as possible in such a circumstance.
> :::

Author: ```suggestion
    Existing applications often default to assuming a `FORM` of "City, County, State, Country",
    and some even ignore any `FORM` substructures and treat payloads with a smaller number of
    elements as if they had additional blank elements at the end. Accordingly, to increase
    the chances of interoperability it is recommended that blank elements be used to ensure
    payloads can be interpreted as best as possible in such a circumstance.
```
